ludwig/combiners/combiners.py

```diff
@@ -482,19 +491,18 @@ class Meta:
         unknown = INCLUDE
 
 
-class TabNetCombiner(CombinerClass):
+@register_combiner(name='tabnet')
+class TabNetCombiner(Combiner):
     def __init__(
             self,
-            input_features: Dict,
+            input_features: Dict[str, "InputFeature"],
```

Comment:
Is wrapping the class name in quotes the standard way to declare a type hint for a Dict input arg?

Reply:
It's a way to reference the type without needing to import it (in this case, it would be a circular dependency).